GERTi/GERTi Client.lua

```diff
@@ -186,30 +208,30 @@ end
 local function receivePacket(eventName, receivingModem, sendingModem, port, distance, code, ...)
     print(code)
     -- Attempt to call a handler function to further process the packet
-    if handler.code ~= nil then
-        return handler.code(eventName, receivingModem, sendingModem, port, distance, code, ...)
+    if handler[code] ~= nil then
+        return handler[code](eventName, receivingModem, sendingModem, port, distance, code, ...)
     else
         return false
     end
 end
 
 -- transmit broadcast to check for neighboring GERTi enabled computers
-if component.isAvailable("tunnel") then
-    tunnel.send("GERTiStart")
-    handleEvent(event.pull(1, "modem_message"))
-end
+--if (tunnel) then
+--    tunnel.send("GERTiStart")
+--    handleEvent(event.pull(1, "modem_message"))
```

Comment:
Please uncomment this block and replace handleEvent(event.pull(1, "modem_message")) with receivePacket(event.pull(1, "modem_message"))